_stories/molecules/SearchMultiSelect/README.md

```diff
@@ -52,30 +62,30 @@ key        | uniquely identifies each options (key must be \`unique\`)
+| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| options     | list of options {array of objects}                                                                                                                                                                                       |
+| update      | callback that returns the latest change on options {function}                                                                                                                                                            |
+| onChange    | callback that returns the current input value, useful for autocompletion {function}                                                                                                                                      |
+| filter      | optional custom function to filter elements, the arguments passed to it are current options {Array} and search term {String}, must return an array with the indexes of the options to display in the dropdown {function} |
+| context     | for now, this only affects the color of the Tag elements. (For not it only affects the tags, but it will support the input itself soon).                                                                                 |
+| placeholder | placeholder text {String}                                                                                                                                                                                                |
+| size        | size for the input, only md or sm are allowed {String}                                                                                                                                                                   |
+| searchKeys  | flag to signal if the search should also match against object keys {Boolean}                                                                                                                                             |
+| multiTerm   | flag to signal if the search should match against multiple terms {Boolean}                                                                                                                                               |
+| termDivider | string or regexp used to divide the search term, defaults to /[ ,]+/ (empty space and comma) {String OR RegExp}                                                                                                          |
+| inputRef    | ref applied to the input, useful for setting focus {function                                                                                                                                                             | object} |
```

Comment:
Table is broken here

Reply:
That's was automatically done by prettier. Looks fine when rendered.
![screenshot_20180801_163555](https://user-images.githubusercontent.com/1082158/43550388-07dd002e-95a9-11e8-9b34-8443ad91b285.png)

Reply:
Oh I'm seeing this:
![screen shot 2018-08-01 at 2 58 36 pm](https://user-images.githubusercontent.com/11967290/43551317-785ceb2e-959b-11e8-90ab-cb7aecda2251.png)

Reply:
OH! my bad, yup, I'll fix it, thanks!

Reply:
Fixed